plugin/adsk/scripts/mayaUsd_createStageFromFile.mel

```diff
@@ -192,5 +192,9 @@ global proc mayaUsd_createStageFromFile() {
     if (size($result) > 0) {
         doCreateStage($result[0]);
     }
-        
+}
+
+global proc mayaUsd_createStageFromFilePath(string $usdFile)
+{
+    doCreateStage($usdFile);
```

Comment:
Simple global proc that allows me to do the same as proc above, but sending in a file path, skipping the file dialog.